packages/rest/src/rest.component.ts

```diff
@@ -94,6 +60,53 @@ export class RestComponent implements Component {
       Object.assign(apiSpec, {servers: config.openApiSpec.servers});
     }
     app.bind(RestBindings.API_SPEC).to(apiSpec);
+    app.server(RestServer);
+    this.bindBodyParsers(app);
```

Comment:
1. As I commented earlier, I believe this is a backwards-incompatible change that requires a semver-major release.

2. Isn't this particular implementation a step backwards? Before, we used declarative approach to export servers and bindings. Now we are moving to imperative style.

Reply:
Reverted